app/build.gradle

```diff
@@ -25,9 +25,14 @@ android {
 dependencies {
     implementation "com.plaid.link:sdk-core:2.1.0"
 
-    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation "androidx.appcompat:appcompat:1.1.0"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.2.0'
-    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    implementation 'com.google.android.material:material:1.1.0'
+    implementation "androidx.core:core-ktx:1.2.0"
+    implementation "androidx.constraintlayout:constraintlayout:1.1.3"
+    implementation "com.google.android.material:material:1.1.0"
+
+    implementation "com.squareup.retrofit2:retrofit:2.9.0"
+    implementation "com.squareup.retrofit2:converter-gson:2.9.0"
+    implementation "com.squareup.retrofit2:adapter-rxjava2:2.9.0"
```

Comment:
we might want to avoid rx and coroutines and just use the default retrofit callback so it's simpler for developers

Reply:
The boilerplate to deal with Call obfuscates the actual logic imo